bindings/python/tools/python/ostk/astrodynamics/utilities.py

```diff
@@ -19,7 +22,152 @@
 from ostk.physics.environment.gravitational import Earth as EarthGravitationalModel
 
 from ostk.astrodynamics import Trajectory
+from ostk.astrodynamics.converters import coerce_to_datetime
 from ostk.astrodynamics.trajectory import State
+from ostk.astrodynamics.trajectory import StateBuilder
+from ostk.astrodynamics.trajectory import Orbit
+from ostk.astrodynamics.trajectory.state.coordinate_subset import CartesianPosition
+from ostk.astrodynamics.trajectory.state.coordinate_subset import CartesianVelocity
+from ostk.astrodynamics.trajectory import LocalOrbitalFrameFactory
+
+
+POSITION_VELOCITY_STATE_BUILDER: StateBuilder = StateBuilder(
+    frame=Frame.GCRF(),
+    coordinate_subsets=[
+        CartesianPosition.default(),
+        CartesianVelocity.default(),
+    ],
+)
+
+DEFAULT_INERTIAL_FRAME: Frame = Frame.GCRF()
+
+
+@dataclass
+class Residual:
```

Comment:
Why are we adding another state representation? I think we should just return list[State]